docs/release-notes/NuGet-6.11.md

```diff
@@ -0,0 +1,101 @@
+ms.date: 8/1/2024
+ms.topic: conceptual
+---
+# NuGet 6.11 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.11**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.11](https://visualstudio.microsoft.com/downloads/) | [8.0.4xx](https://dotnet.microsoft.com/download/dotnet/8.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
+
+## Summary: What's New in 6.11
+
+* Suppress NuGetAudit warnings for specific advisories for PackageReference projects - [#13679](https://github.com/NuGet/Home/issues/13679)
+
+* Add `--allow-insecure-connections` option to dotnet SDK - [#13396](https://github.com/NuGet/Home/issues/13396)
+
+* Swap authors for owners in Visual Studio Package Manager UI - [#12501](https://github.com/NuGet/Home/issues/12501)
+
+* [Feature]: add dotnet nuget why to dotnet CLI - [#11943](https://github.com/NuGet/Home/issues/11943)
+
+* NuGet cannot restore from HTTPS sources that have SSL certificate problems - [#4387](https://github.com/NuGet/Home/issues/4387)
+
+### Breaking changes
+
+* `MSBuildRestoreUtility.GetRestoreAuditProperties` needs a breaking change to read `NuGetAuditSuppress` items - [#13313](https://github.com/NuGet/Home/issues/13313)
+
+* NuGetAudit should check transitive packages by default when the .NET 9 SDK is installed - [#13293](https://github.com/NuGet/Home/issues/13293)
+
+### Issues fixed in this release
+
+* IVsNuGetProjectUpdateEvents.ProjectUpdateStarted or ProjectUpdateFinished should only provide a list of files that will get changed. - [#13413](https://github.com/NuGet/Home/issues/13413)
+
+* `dotnet` commands should not output usage information on all errors - [#13251](https://github.com/NuGet/Home/issues/13251)
+
+* [DCR]: NuGet causes many ArgumentExceptions to be thrown & caught in VS - [#11535](https://github.com/NuGet/Home/issues/11535)
+
+* Remove .NET Framework TFM from NuGet.CommandLine.XPlat - [#8452](https://github.com/NuGet/Home/issues/8452)
+
+* PERF: GetPackageInfo allocates by unnecessarily calling GetNupkgMetadataPath - [#13556](https://github.com/NuGet/Home/issues/13556)
+
+* 'dotnet nuget why' crashes when using --framework option - [#13547](https://github.com/NuGet/Home/issues/13547)
+
+* SignatureUtility.GetCertificates in NuGet.Client can skip calling Dispose on error - [#13535](https://github.com/NuGet/Home/issues/13535)
+
+* 'dotnet nuget why' does not work when a directory is provided for the 'Path' argument - [#13527](https://github.com/NuGet/Home/issues/13527)
+
+* NuGet IntelliCode Package Suggestions are missing Author in packages list - [#13515](https://github.com/NuGet/Home/issues/13515)
+
+* Rebuilding in VS causes unnecessary restores - [#13505](https://github.com/NuGet/Home/issues/13505)
+
+* MSB4181: The "Restore Task" task returned false but did not log an error. - [#13460](https://github.com/NuGet/Home/issues/13460)
```

Comment:
The commit that closed this issue is https://github.com/NuGet/NuGet.Client/commit/de9c314080687bfe4da4baf4b3d1a9d38b6ace3a

But this commit was reverted: https://github.com/NuGet/NuGet.Client/commit/81b4565544b0f3a90f4527b550ff3318104b1370

What's the status? Did we fix "the restore task returned false but did not log an error"?

Reply:
looks like the revert was not part of any branch (probably a private branch that was deleted). @nkolev92 can you confirm?

Reply:
> looks like the revert was not part of any branch (probably a private branch that was deleted). @nkolev92 can you confirm?

It's not. 
Just some testing.